File: app/src/main/java/com/example/desafio_concrete/networkUtils/Owner.java
package com.example.desafio_concrete.networkUtils;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Owner {

    @SerializedName("login")
    @Expose
    private String nameAuthor;

    @SerializedName("avatar_url")
    @Expose
    private String avatarUrl;


    public String getNameAuthor() {
        return nameAuthor;
    }

    public void setNameAuthor(String nameAuthor) {
        this.nameAuthor = nameAuthor;
    }

    public String getAvatarUrl() {
        return avatarUrl;
    }

    public void setAvatarUrl(String avatarUrl) {
        this.avatarUrl = avatarUrl;
    }

    @Override
    public String toString() {
        return "Owner{" +
                "login='" + nameAuthor + '\'' +
                ", avatar_url='" +avatarUrl + '\'' +
                "}";
    }
}

File: app/src/main/java/com/example/desafioandroid/schema/Owner.kt
package com.example.desafioandroid.schema

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Owner {
    @SerializedName("login")
    @Expose
    var login: String? = null
    @SerializedName("id")
    @Expose
    var id: Int? = null
    @SerializedName("avatar_url")
    @Expose
    var avatarUrl: String? = null

}
